### PR TITLE
Irminger deploy csv updates/corrections

### PR DESCRIPTION
**Pull request by rtravis-whoi**
SUMO-10 - added note for SWND2 being raised to same height as FDCHP

Updated the following based on written deploy/recover logs (QLR tables have been updated and are attached).  Written logs are in Vault: Irminger 10 Deployment/Logs folder.  Also updated a few OSNAP CTDMO S/Ns on FLMA & FLMB (700m, 400m, 100m on both - confirmed via photos) based on written config checks while at sea (written scans are attached - I'm still working on updating the ones in Vault but I did double-check everything):
SUMO-9 recover time 10:12:00
HYPM-10 - longitude (used sig figs in written log surveyed location to calculate)
FLMA-10 - deploy time 14:56:00; recover time midwater 10:00:00 and anchor 11:29:00; updates to CTDMO S/Ns
FLMB-10 - longitude (used sig figs in written log surveyed location to calculate); updates to CTDMO S/Ns

[config_scans.zip](https://github.com/WHOIGit/ooicgsn-asset-management/files/12887187/config_scans.zip)
[3202-01002_tablesupdated.pdf](https://github.com/WHOIGit/ooicgsn-asset-management/files/12887192/3202-01002_tablesupdated.pdf)

### CODE REVIEW

File: deployment/GI01SUMO_Deploy.csv [hunk 1]
```diff
@@ -627,5 +627,5 @@ AR76-03,,,,GI01SUMO-SBD12-06-METBKA003,10,1,2023-08-31T16:00:00,,CGMGI-01SUMO-00
 AR76-03,,,,GI01SUMO-SBD12-06-METBKA004,10,1,2023-08-31T16:00:00,,CGMGI-01SUMO-00010,,CGINS-METLWR-00244,59.944783,-39.5736,,-5,2663,,,
 AR76-03,,,,GI01SUMO-SBD12-06-METBKA005,10,1,2023-08-31T16:00:00,,CGMGI-01SUMO-00010,,CGINS-METPRC-00264,59.944783,-39.5736,,-5,2663,,,
 AR76-03,,,,GI01SUMO-SBD12-06-METBKA006,10,1,2023-08-31T16:00:00,,CGMGI-01SUMO-00010,,CGINS-METSWR-00253,59.944783,-39.5736,,-5,2663,,,
-AR76-03,,,,GI01SUMO-SBD12-06-METBKA007,10,1,2023-08-31T16:00:00,,CGMGI-01SUMO-00010,,CGINS-METWND-00258,59.944783,-39.5736,,-6,2663,,,
+AR76-03,,,,GI01SUMO-SBD12-06-METBKA007,10,1,2023-08-31T16:00:00,,CGMGI-01SUMO-00010,,CGINS-METWND-00258,59.944783,-39.5736,,-6,2663,sensor raised to same height as FDCHP,,
```
Comment (cooleyky) [inline deployment/GI01SUMO_Deploy.csv:630]: Looks good. Updated recovery time matches QLR.

File: deployment/GI02HYPM_Deploy.csv [hunk 2]
```diff
@@ -76,12 +76,12 @@ AR69-01,,AR76-03,,GI02HYPM-WFP02-04-CTDPFL000,9,1,2022-07-01T12:49:00,2023-09-06
...
+AR76-03,,,,GI02HYPM-WFP02-00-WFPENG000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,CGVEH-GWFPGX-45201,CGCON-GWFPGX-45201,59.973718,-39.52523,,2560,2661,,,D
+AR76-03,,,,GI02HYPM-WFP02-01-FLORDL000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,CGVEH-GWFPGX-45201,CGINS-FLORDL-04013,59.973718,-39.52523,,2560,2661,,,D
+AR76-03,,,,GI02HYPM-WFP02-03-DOSTAL000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,CGVEH-GWFPGX-45201,CGINS-DOSTAL-02198,59.973718,-39.52523,,2560,2661,,,D
+AR76-03,,,,GI02HYPM-WFP02-04-CTDPFL000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,CGVEH-GWFPGX-45201,CGINS-CTDPFL-00150,59.973718,-39.52523,,2560,2661,,,D
+AR76-03,,,,GI02HYPM-WFP02-05-VEL3DL000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,CGVEH-GWFPGX-45201,CGINS-VEL3DL-01256,59.973718,-39.52523,,2560,2661,,,D
+AR76-03,,,,GI02HYPM-MPM01-02-ZPLSGA009,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,,CGINS-ZPLSGA-55092,59.973718,-39.52523,,150,2661,upward-looking,,D
+AR76-03,,,,GI02HYPM-MPM01-02-ZPLSGA010,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,,CGINS-ZPLSGA-55093,59.973718,-39.52523,,150,2661,downward-looking,,D
```
Comment (cooleyky) [inline deployment/GI02HYPM_Deploy.csv:87]: Looks good

File: deployment/GI03FLMA_Deploy.csv [hunk 3]
```diff
@@ -207,55 +207,55 @@ AR60-01,,AR69-01,,GI03FLMA-RIS01-00-SIOENG000,8,1,2021-08-13T13:22:00,2022-07-08
...
+AR76-03,,,,GI03FLMA-RIM01-02-CTDMOG048,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-CTDMOG-13590,59.7711,-39.883083,,500,2691,,,C
+AR76-03,,,,GI03FLMA-RIM01-02-CTDMOH049,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-CTDMOH-13633,59.7711,-39.883083,,750,2691,,,C
+AR76-03,,,,GI03FLMA-RIM01-02-CTDMOH050,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-CTDMOH-12219,59.7711,-39.883083,,1000,2691,,,C
+AR76-03,,,,GI03FLMA-RIM01-02-CTDMOH051,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-CTDMOH-13447,59.7711,-39.883083,,1500,2691,,,C
+AR76-03,,,,GI03FLMA-RIS01-00-SIOENG000,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGCON-SIOCON-13283,59.7711,-39.883083,,30,2691,,,C
+AR76-03,,,,GI03FLMA-RIS01-03-DOSTAD000,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-DOSTAD-00224,59.7711,-39.883083,,30,2691,,,C
+AR76-03,,,,GI03FLMA-RIS01-04-PHSENF000,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-PHSENF-P0141,59.7711,-39.883083,,30,2691,,,C
+AR76-03,,,,GI03FLMA-RIS01-05-FLORTD000,10,1,2023-09-05T14:56:00,,CGMGI-03FLMA-00010,,CGINS-FLORTD-01299,59.7711,-39.883083,,30,2691,,,C
```
Comment (cooleyky) [inline deployment/GI03FLMA_Deploy.csv:261]: Looks good

File: deployment/GI03FLMB_Deploy.csv [hunk 4]
```diff
@@ -233,29 +233,29 @@ AR69-01,,AR76-03,,GI03FLMB-RIS01-00-SIOENG000,9,1,2022-07-02T13:43:00,2023-09-09
...
+AR76-03,,,,GI03FLMB-RIM01-02-CTDMOG068,10,1,2023-09-06T15:53:00,,CGMGI-03FLMB-00010,,CGINS-CTDMOG-13623,59.719785,-39.353795,,500,2820,,,C
+AR76-03,,,,GI03FLMB-RIM01-02-CTDMOH069,10,1,2023-09-06T15:53:00,,CGMGI-03FLMB-00010,,CGINS-CTDMOH-13631,59.719785,-39.353795,,750,2820,,,C
+AR76-03,,,,GI03FLMB-RIM01-02-CTDMOH070,10,1,2023-09-06T15:53:00,,CGMGI-03FLMB-00010,,CGINS-CTDMOH-13454,59.719785,-39.353795,,1000,2820,,,C
+AR76-03,,,,GI03FLMB-RIM01-02-CTDMOH071,10,1,2023-09-06T15:53:00,,CGMGI-03FLMB-00010,,CGINS-CTDMOH-13457,59.719785,-39.353795,,1500,2820,,,C
+AR76-03,,,,GI03FLMB-RIS01-00-SIOENG000,10,1,2023-09-06T15:53:00,,CGMGI-03FLMB-00010,,CGCON-SIOCON-13989,59.719785,-39.353795,,30,2820,,,C
+AR76-03,,,,GI03FLMB-RIS01-03-DOSTAD000,10,1,2023-09-06T15:53:00,,CGMGI-03FLMB-00010,,CGINS-DOSTAD-00226,59.719785,-39.353795,,30,2820,,,C
+AR76-03,,,,GI03FLMB-RIS01-04-PHSENF000,10,1,2023-09-06T15:53:00,,CGMGI-03FLMB-00010,,CGINS-PHSENF-P0142,59.719785,-39.353795,,30,2820,,,C
+AR76-03,,,,GI03FLMB-RIS01-05-FLORTD000,10,1,2023-09-06T15:53:00,,CGMGI-03FLMB-00010,,CGINS-FLORTD-01322,59.719785,-39.353795,,30,2820,,,C
```
Comment (cooleyky) [inline deployment/GI03FLMB_Deploy.csv:261]: Looks good.

**Review by cooleyky — APPROVED** [review]
Looks good. Matches information in provided logs.

**Comment by rtravis-whoi** [comment]
[Surface Mooring Wind Sensor Heights [DT-214]](https://trello.com/c/KavGDFE6/214-surface-mooring-wind-sensor-heights-dt-214)

File: deployment/GI02HYPM_Deploy.csv [hunk 5]
```diff
@@ -76,12 +76,12 @@ AR69-01,,AR76-03,,GI02HYPM-WFP02-04-CTDPFL000,9,1,2022-07-01T12:49:00,2023-09-06
...
+AR76-03,,,,GI02HYPM-WFP02-00-WFPENG000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,CGVEH-GWFPGX-45201,CGCON-GWFPGX-45201,59.973718,-39.525223,,2560,2661,,,D
+AR76-03,,,,GI02HYPM-WFP02-01-FLORDL000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,CGVEH-GWFPGX-45201,CGINS-FLORDL-04013,59.973718,-39.525223,,2560,2661,,,D
+AR76-03,,,,GI02HYPM-WFP02-03-DOSTAL000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,CGVEH-GWFPGX-45201,CGINS-DOSTAL-02198,59.973718,-39.525223,,2560,2661,,,D
+AR76-03,,,,GI02HYPM-WFP02-04-CTDPFL000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,CGVEH-GWFPGX-45201,CGINS-CTDPFL-00150,59.973718,-39.525223,,2560,2661,,,D
+AR76-03,,,,GI02HYPM-WFP02-05-VEL3DL000,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,CGVEH-GWFPGX-45201,CGINS-VEL3DL-01256,59.973718,-39.525223,,2560,2661,,,D
+AR76-03,,,,GI02HYPM-MPM01-02-ZPLSGA009,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,,CGINS-ZPLSGA-55092,59.973718,-39.525223,,150,2661,upward-looking,,D
+AR76-03,,,,GI02HYPM-MPM01-02-ZPLSGA010,10,1,2023-09-04T18:01:00,,CGMGI-02HYPM-00010,,CGINS-ZPLSGA-55093,59.973718,-39.525223,,150,2661,downward-looking,,D
```
Comment (cooleyky) [inline deployment/GI02HYPM_Deploy.csv:87]: Looks good. Decimal degrees match converted longitude from config sheet.

**Review by cooleyky — APPROVED** [review]
Looks good!

File: deployment/GI01SUMO_Deploy.csv (hunk 1 above)
Comment (sauuyer) [reply deployment/GI01SUMO_Deploy.csv:630]: fdchp height change edit also looks good

File: deployment/GI02HYPM_Deploy.csv (hunk 5 above)
Comment (sauuyer) [reply deployment/GI02HYPM_Deploy.csv:87]: my calculated longitude is -39.52523 from 39 31.5138 (value in config sheet and qlr).

File: deployment/GI03FLMA_Deploy.csv (hunk 3 above)
Comment (sauuyer) [reply deployment/GI03FLMA_Deploy.csv:261]: new midwater and anchor recovery times look good for flma9. deployment time for flma10 looks good. ctdmoh updates look good based on what is in the config sheet for flma10.

File: deployment/GI03FLMB_Deploy.csv (hunk 4 above)
Comment (sauuyer) [reply deployment/GI03FLMB_Deploy.csv:261]: lon updates look good based on converted value from qlr. updated ctdmoh values look good based on revised config sheet.

**Review by sauuyer — COMMENTED** [review]
Everything looks good except potentially one of the HYMP values (left a comment on that sheet)

**Review by sauuyer — APPROVED** [review]

**Pull request merged**